R/derive_var_confirmation_flag.R

```diff
@@ -0,0 +1,350 @@
+#' Derive Confirmation Flag
+#'
+#' Derive a flag which depends on subsequent observations of the dataset. For
```

Comment:
change "subsequent" to "other" as they could use `before` option

Reply:
Good point! Updated